man/coredump.conf.xml

```diff
@@ -86,7 +86,8 @@
         <listitem><para>The maximum size in bytes of a core
         which will be processed. Core dumps exceeding this size
         may be stored, but the backtrace will not be generated.
-        </para>
+        Like other sizes in this same config file, the usual
+        suffixes are allowed (B, K, M, G, T, P, and E.)</para>
```

Comment:
i think this still should say "to the base of 1024" somewhere.

Reply:
Done! (And squash/rebase)